src/intrinsics.cpp

```diff
@@ -1474,8 +1474,6 @@ static inline uint16_t float_to_half(float param)
     return h;
 }
 
-#if !defined(_OS_DARWIN_)   // xcode already links compiler-rt
-
```

Comment:
@KristofferC I'm not sure if you want to merge this commit into your backport or not; it's up to you.